docs/static/governance/avm-standard-github-labels.csv

```diff
@@ -0,0 +1,26 @@
+Name,Description,HEX
```

Comment:
I'm not suggesting changing this now, but I think it might have been easier to store the labels in the native GH API format (JSON). It would have been easier to work with the Hugo bits too as we could have defined a struct and used json tags foe the struct fields.

https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#list-labels-for-a-repository

E.g. 

```json
[
  {
    "name": "",
    "description": "",
    "color": ""
  }
]
```

Could have then sent each array member directly to GH using the `gh api`... command.

Reply:
We did think about this, and can re-consider in the future, but CSV is more accessible to non-devs to be able to interact with. We have the same with the central JSON file.

Maybe we can auto-compile a CSV and JSON either way to make both accessible?

Future though with current priorities